src/gluonts/torch/model/predictor.py

```diff
@@ -83,3 +79,58 @@ def predict(
                 output_transform=self.output_transform,
                 num_samples=num_samples,
             )
+
+    def serialize(self, path: Path) -> None:
+        super().serialize(path)
+
+        # serialize network
+        with (path / f"prediction_net.json").open("w") as fp:
+            print(dump_json(self.prediction_net), file=fp)
+        torch.save(
+            self.prediction_net.state_dict(), path / "prediction_net_state"
+        )
+
+        # serialize transformation chain
+        with (path / "input_transform.json").open("w") as fp:
+            print(dump_json(self.input_transform), file=fp)
+
+        # FIXME: also needs to serialize the output_transform
```

Comment:
@vafl do you remember what the idea behind `output_transform` was? I don't think this is used anywhere (and also MXNet-based predictors have this TODO, so they don't serialize it)

Reply:
It is some sort of invertible transformation that can be put into the transformation chain. This will not yield the correct jacobian, but some transformations are hard to implement inside the graph (for instance, the empirical cdf of `GPVAR`).

Reply:
Ok -- I think I left this FIXME here since the MXNet-based predictor classes have this

Reply:
Sounds good.